specification/configuration.md

```diff
@@ -89,6 +90,52 @@ environment variables:
   If `SPLUNK_CONFIG` is defined then these options are not required but MAY be
   defined.
 
+### Kubernetes
+
+While Kubernetes supports container technology that can be configured using
+environment variables, package management solutions such as Helm charts and
+Operators require YAML-based configuration. As a result, Kubernetes package
+management solutions MUST support the following YAML configuration options:
+
+- `clusterName` ()             : [REQUIRED] Name of the cluster.
+- `cloudProvider` ()           : Where Kubernetes is deployed.
+- `distribution` ()            : Which distribution of Kubernetes is deployed.
+- `environment` ()             : Name of the environment; if not defined then skipped.
+- `agent`                      : Deployed as a DaemonSet.
+  - `enabled` (`true`)         : Whether agent is deployed.
+  - `config` ()                : Updates configuration. Non-list options merged, list options override.
+- `gateway`                    : Deployed as a clustered Service and receives data from agent.
+  - `enabled` (`false`)        : Whether gateway is deployed.
+  - `config` ()                : Updates configuration. Non-list options merged, list options override.
+- `clusterReceiver`            : Deployed as a single replica deployment and collects Kubernetes API cluster and event telemetry.
+  - `enabled` (`true`)         : Whether k8sClusteReceiver is deployed. Ignored if `metricsEnabled` is `false`.
+  - `config` ()                : Updates configuration. Non-list options merged, list options override.
+
+In addition, at least one of the following sections MUST be specified:
```

Comment:
"following sections" is a bit vague, it can be interpreted as a paragraph or markdown section as well as a level in the following list. Can we nail this down?

```suggestion
In addition, at least one of the the following configuration groups, `splunkObservability` or `splunkPlatform`, MUST be specified:
```